hermes-console-vue/src/views/stats/StatsView.vue

```diff
@@ -0,0 +1,115 @@
+<script async setup lang="ts">
+  import { formatNumber } from '@/utils/number-formatter/number-formatter';
+  import { useI18n } from 'vue-i18n';
+  import { useStats } from '@/composables/use-stats/useStats';
+  import ConsoleAlert from '@/components/console-alert/ConsoleAlert.vue';
+  import LoadingSpinner from '@/components/loading-spinner/LoadingSpinner.vue';
+
+  const { t } = useI18n();
+
+  const { stats, loading, error } = useStats();
+</script>
+
+<template>
+  <v-container>
+    <v-row v-if="loading || error" justify="center" cols="12">
+      <v-col md="12">
+        <loading-spinner v-if="loading" />
+        <console-alert
+          v-if="error.fetchError"
+          :title="t('stats.connectionError.title')"
+          :text="t('stats.connectionError.text')"
+          type="error"
+        />
+      </v-col>
+    </v-row>
+    <div v-if="stats">
+      <v-row class="mt-16">
+        <h1>{{ $t('stats.topics') }}</h1>
+      </v-row>
+      <v-row>
+        <v-col md="3">
+          <v-row>
+            <h2>{{ $t('stats.total') }}</h2>
```

Comment:
One last little thing. We can use headings classes from vuetify typography: https://vuetifyjs.com/en/styles/text-and-typography/. This way we would have consistent headings accros our app. Other than that: Looks pretty good! Congrats on your first vue 3 PR :D

Reply:
fixed! and thanks :)